### PR TITLE
Disabled autofill/autocomplete for name field

### PR DESCRIPTION
**Pull request by sam9111**
chrome-capture

### CODE REVIEW

File: src/Components/Patient/PatientRegister.tsx [hunk 1]
```diff
@@ -1057,6 +1057,9 @@ export const PatientRegister = (props: PatientRegisterProps) => {
                       value={state.form.name}
                       onChange={handleChange}
                       errors={state.errors.name}
+                      inputProps={{
+                        autoComplete: "new-password",
+                      }}
```
Comment (bodhish) [inline src/Components/Patient/PatientRegister.tsx]: 🤔 How is this fixing the issue?

Comment (sam9111) [reply src/Components/Patient/PatientRegister.tsx]: Found the solution for this issue here: https://material-ui.com/components/autocomplete/

**Comment by nihal467** [comment]
@bodhish can I test it..?

**Comment by sam9111** [comment]
Strangely "no" fixes the issue while "off" did nothing. Using Brave(chrome based browser) @nihal467 @bodhish

**Comment by samholics** [comment]
tested, verified

autofill not coming up now.

**Review by bodhish — APPROVED** [review]

**Pull request merged**